src/test/java/com/google/devtools/build/lib/actions/ActionCacheCheckerTest.java

```diff
@@ -616,6 +624,30 @@ public void saveOutputMetadata_localMetadataIsSameAsRemoteMetadata_cached() thro
     assertThat(entry.getOutputFile(output)).isEqualTo(createRemoteFileMetadata(content));
   }
 
+  @Test
+  public void saveOutputMetadata_localMetadataIsSameAsRemoteMetadata_withTargetExecPath_cached()
```

Comment:
I would recommend parameterizing the test case above: `@TestParameter({"null", "/target/path"}} String symlinkTargetPath)`.